src/CameraWidgetForPhoneGap/widget/CameraWidgetForPhoneGap.js

```diff
@@ -253,6 +254,17 @@ require([
                     }
                 });
             }
-        }
+
+            if (this.onSaveNanoflow && this.mxcontext) {
+                mx.data.callNanoflow({
+                    nanoflow: this.onSaveNanoflow,
+                    origin: this.mxform,
+                    context: this.mxcontext,
+                    error: function (error) {
+                        mx.ui.error(error.message);

```

Comment:
Add better description. `"Error executing nanoflow " + this.onSaveNanoflow + " : " + error.message`

Reply:
Message improved